### PR TITLE
2018 Community Survey Blog Post and Callout

### PR DESCRIPTION
**Pull request by akankshadharkar**
blogpost
callout

Futher changes to be made:
- Content of the 2018 blog post draft on the [/ember-community-survey-2018](https://github.com/akankshadharkar/website/blob/4a572666bada80a177adaf870601453cf202fdde/source/blog/2018-02-18-2018-ember-community-survey.html.md).
- Add the respective 2018 Community Survey and EmberConf logos. 
- Change the shades of orange and blue.
- Update all links to the /ember-community-survey-2018 to direct to the appropriate page.

### CODE REVIEW

File: source/_survey-reminder.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% if !current_page.data.no_survey_reminder %>
+    <div class="survey-reminder-wrapper">
+      <div class="survey-reminder-container">
+        Take the <strong>
+        <% link_to 'https://embersurvey.typeform.com/to/tKfRLF' do %>
```
Comment (iezer) [inline source/_survey-reminder.html.erb]: This link should go to `'/ember-community-survey-2018'`

Similar to last year's PR here: https://github.com/emberjs/website/pull/2822/files#diff-54692326d4e8adb021dbd8f90ba70f64R5

File: source/blog/2017-02-27-2017-ember-community-survey.html.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: "2017 Ember Community Survey"
```
Comment (iezer) [inline source/blog/2017-02-27-2017-ember-community-survey.html.md]: There's no need to rename the 2017 blog post file.

**Comment by iezer** [comment]
the branch probably needs to be rebased off the latest master as well

File: source/blog/2017-02-27-2017-ember-community-survey.html.md (hunk 2 above)
Comment (akankshadharkar) [reply source/blog/2017-02-27-2017-ember-community-survey.html.md]: My bad, was changing it in a different file. Accidentally made the change in that file.

**Comment by akankshadharkar** [comment]
2018html
@iezer I have added the screenshot of the `ember-community-survey-2018.html.erb`, as requested.

**Comment by sivakumar-kailasam** [comment]
@akankshadharkar You can use https://ember-website-staging-pr-3199.herokuapp.com/ to view your changes. As you push changes to your branch the changes will get deployed to that site.

**Pull request merged**